ci-operator/config/openshift/cincinnati/openshift-cincinnati-master-presubmit.yaml

```diff
@@ -0,0 +1,32 @@
+base_images:
```

Comment:
This file shouldn't exist.

Reply:
Isn't it used for PRs?

Reply:
no, PRs and merges are both handled from openhsift-cincannit-master.  The presubmits jobs are generated from the single config.  We have one file per repo per branch.

Reply:
Alright, removed